xml/depl_cisco_opflex.xml

```diff
@@ -0,0 +1,686 @@
+      <imageobject role="html">
+       <imagedata fileref="depl_opflex_1.png" width="75%" format="png"/>
+      </imageobject>
+     </mediaobject>
+    </figure>
+</sect1>
+
+    <sect1 xml:id="sec.networking">
+  <title>Configure Networking on Compute Nodes</title>     
+    <para>
+In the example network we have two compute nodes that are connected to our Cisco ACI fabric. Network configuration will be automated in future releases, but for now you must follow these steps to configure networking on all of your compute nodes.
+    </para>
+    
+<procedure>
+    <step>
+        <para>
+            Enter <literal>YAST | System | Network Settings</literal> (Click Continue when you see the Chef client warning).
+    </para>
+   </step> 
+   
+   <step>
+    <para>
+        Highlight the Ethernet device connected to the ACI fabric and press F4 (Edit).
+    </para>
+   </step> 
+   
+    <step>
+    <para>
+        Enable <literal>No Link and IP Setup</literal>.
+    </para>
+   </step> 
+   
+   <step>
+    <para>
+       Go to the General tab and set the MTU to 1600.
+    </para>
+   </step> 
+   
+    <step>
+    <para>
+       Press F10 to continue.
+    </para>
+   </step> 
+</procedure>
+
+
+<para>
+    You should be back at the first screen at <literal>YAST | System | Network Settings</literal>. Press F3 (Add) to open the Hardware Dialog. 
+</para>
+
+  <procedure>
+   <step>
+    <para>
+      Under <literal>Device Type</literal> select VLAN.
+    </para>
+   </step> 
+   
+   <step>
+    <para>
+        Set the <literal>Configuration Name</literal> to match what has been defined in the ACI fabric. In the example network that is <literal>vlan.4093</literal>.
+    </para>
+   </step> 
+   
+    <step>
+    <para>
+        Press F10 (Next) to return to the <literal>Network Card Setup | Address screen.</literal>
+    </para>
+   </step> 
+   
+   <step>
+    <para>
+       Select the Ethernet device that is connected to the ACI fabric  for <literal>Real Interface for VLAN</literal>.
+    </para>
+   </step> 
+   
+    <step>
+    <para>
+       Enter your VLAN ID, which in the example network is 4093.
+    </para>    
+   </step> 
+    
+   <step>
+    <para>
+      Enable <literal>Dynamic Address - DHCP</literal>.
+    </para>
+   </step> 
+   
+    <step>
+    <para>
+      Change to the General tab and set the MTU to 1600.
+    </para>
+   </step> 
+
+    <step>
+    <para>
+      Press F10 to save changes for new VLAN device, and press F10 again to exit Network Settings.
+    </para>
+   </step>    
+</procedure>
+
+
+<para>
+   Now that you have made these changes, re-enter Network Settings and configure routing. 
+</para>
+
+<procedure>
+   <step>
+    <para>
+     Go back into Network Settings (Click Continue when you see the Chef client warning).
+    </para>
+   </step> 
+   
+   <step>
+    <para>
+        Change to the Routing tab and add these entries to the Routing Table:
+        
+        <command>Destination     224.0.0.0
```

Comment:
...and make sure to never use &lt;screen> as an inline object__always use it as a block element.

    Wrong
    <para><screen>...</screen></para>

    Correct:
    <para>...</para>
    <screen>...</screen>